core/blockchain/impl/storage_util.hpp

```diff
@@ -32,9 +32,6 @@ namespace kagome::blockchain {
       // body of the block (extrinsics)
       BLOCK_DATA = 5,
 
-      // justification of the finalized block
-      JUSTIFICATION = 6,
-
```

Comment:
Existing records of justification will be removed? I worry about orphan data

Reply:
This field was not used.

Reply:
Yes, second one. If you sure this code is not used, it is not problem.